src/content/learn/start-a-new-react-project.md

```diff
@@ -1,110 +1,112 @@
 
-**React frameworks on this page solve problems like these by default, with no extra work from your side.** They let you start very lean and then scale your app with your needs. Each React framework has a community, so finding answers to questions and upgrading tooling is easier. Frameworks also give structure to your code, helping you and others retain context and skills between different projects. Conversely, with a custom setup it's easier to get stuck on unsupported dependency versions, and you'll essentially end up creating your own framework—albeit one with no community or upgrade path (and if it's anything like the ones we've made in the past, more haphazardly designed).
+**কোন অতিরিক্ত কাজ ছাড়াই এই পৃষ্ঠার React ফ্রেমওয়ার্ক গুলো ডিফল্টভাবে এই ধরনের সমস্যাগুলি সমাধান করে।** তারা আপনাকে খুব সহজে শুরু করতে সাহায্য করে এবং তারপরে আপনার প্রয়োজনের সাথে আপনার অ্যাপটি স্কেল করতে সাহায্য করে। প্রতিটি React ফ্রেমওয়ার্কের একটি কমিউনিটি রয়েছে, তাই প্রশ্নের উত্তর খুঁজে পাওয়া এবং টুলিং আপগ্রেড করা সহজ। ফ্রেমওয়ার্কগুলি আপনার কোডের কাঠামোও দেয়, আপনাকে এবং অন্যদেরকে বিভিন্ন প্রকল্পের মধ্যে প্রসঙ্গ এবং দক্ষতা বজায় রাখতে সাহায্য করে। বিপরীতভাবে, একটি কাস্টম সেটআপের মধ্যে খুব সহজেই unsupported dependency version গুলিতে আটকে যাওয়া সহজ, এবং আপনি মূলত আপনার নিজস্ব কাঠামো তৈরি করতে পারবেন—যদিও কোনো framework আপগ্রেড পথ নেই (এবং যদি এটি আরও এলোমেলোভাবে ডিজাইন করা আমাদের অতীতে তৈরি করা framework গুলির মতো কিছু হয়)।
 
-If you're still not convinced, or your app has unusual constraints not served well by these frameworks and you'd like to roll your own custom setup, we can't stop you--go for it! Grab `react` and `react-dom` from npm, set up your custom build process with a bundler like [Vite](https://vitejs.dev/) or [Parcel](https://parceljs.org/), and add other tools as you need them for routing, static generation or server-side rendering, and more.
+আপনি যদি এখনও নিশ্চিত না হন, বা আপনার অ্যাপে অস্বাভাবিক সীমাবদ্ধতা থাকে যা এই framework গুলো দ্বারা ভালভাবে পরিবেশিত না হয় এবং আপনি আপনার নিজস্ব কাস্টম সেটআপ রোল করতে চান, আমরা আপনাকে থামাব না--এটির জন্য npm থেকে `react` এবং `react-dom` ইন্সটল করুন, [Vite](https://vitejs.dev/) or [Parcel](https://parceljs.org/) এর মতো একটি বান্ডলার দিয়ে আপনার কাস্টম বিল্ড প্রক্রিয়া সেট আপ করুন এবং রাউটিং, স্ট্যাটিক জেনারেশন বা সার্ভার-সাইড রেন্ডারিং এবং আরও অনেক কিছুর জন্য আপনার প্রয়োজন অনুযায়ী অন্যান্য টুল যোগ করুন।
```

Comment:
If you're still not convinced => আপনি যদি এখনো ভরসা না পান
এই ট্রান্সলেশন টা কেমন?